### PR TITLE
Allow yum packages to be installed

### PR DESCRIPTION
**Pull request by dries007**
Added `system-packages` property that gets passed to `yum install -y`.

This allows me to install libraries my C extention needs.

Another option would be to allow arbitrary commands before lauch, but I don't know what would be a good way of supporting that, so I did it this way.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -11,6 +11,10 @@ inputs:
     description: 'Packages required at build time, space-separated'
```
Comment (RalfG) [inline action.yml]: For clarity, we should specify this option as "pip packages". This also needs to be changed in `README.md`
```suggestion
    description: 'pip packages required at build time, space-separated'
```

File: entrypoint.sh [hunk 2]
```diff
@@ -4,6 +4,11 @@
 # CLI arguments
 PY_VERSIONS=$1
 BUILD_REQUIREMENTS=$2
+SYSTEM_REQUIREMENTS=$3
+
+if [ ! -z "$SYSTEM_REQUIREMENTS" ]; then
+    yum install -y ${SYSTEM_REQUIREMENTS}
+fi
```
Comment (RalfG) [inline entrypoint.sh]: The option is called `system-packages` in action.yml. We should keep this naming consistent in the entrypoint.sh. `SYSTEM_REQUIREMENTS` should be changed to `SYSTEM_PACKAGES`.

File: entrypoint.sh [hunk 3]
```diff
@@ -4,6 +4,11 @@
 # CLI arguments
 PY_VERSIONS=$1
 BUILD_REQUIREMENTS=$2
+SYSTEM_REQUIREMENTS=$3
+
+if [ ! -z "$SYSTEM_REQUIREMENTS" ]; then
+    yum install -y ${SYSTEM_REQUIREMENTS}
```
Comment (RalfG) [inline entrypoint.sh]: Can you also add an error message and exit code if the yum install failed?
```suggestion
    yum install -y ${SYSTEM_REQUIREMENTS}  || { echo "Installing yum package(s) failed."; exit 1; }
```

**Review by RalfG — CHANGES_REQUESTED** [review]
Thanks for the addition of this option! Just a few things need to be changed for consistency and clarity between the options.
Can you also add the additional option and changes to README.md?

**Comment by dries007** [comment]
Will do

**Comment by dries007** [comment]
I made the suggested changes.

Just an FIY for anyone trying to find out what packages to include, it's easiest to spin up a docker instance with `docker run -it 'quay.io/pypa/manylinux1_x86_64' bash` and using `yum` from there.
Most of the online package databases have dropped support for the CentOS 5.

**Review by RalfG — APPROVED** [review]
Thanks!

**Pull request merged**